blocks.md

```diff
@@ -577,7 +577,7 @@ Show detailed weather in San Francisco through the OpenWeatherMap service:
 ```toml
 [[block]]
 block = "weather"
-format = "{weather} ({location}) {temp}°, {wind} km/s"
+format = "{weather} ({location}) {temp}°, {wind} km/s {direction}"
```

Comment:
I believe this should be `m/s`, as you pointed out in #261.

Reply:
Done.